api/trades/TradesApi.ts

```diff
@@ -12,6 +14,7 @@ export class TradesApi {
     }
 
     buy(trade: Trade) {
+        // datastore.set(trade); // TODO enable this
         this.buyorders.push(trade);
```

Comment:
We would remove the `buyorders` array from this class (potentially add it back in again as a caching layer).